userguide/content/en/blog/2024/0.10.0.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Docsy 0.10.0 release report
+linkTitle: Release 0.10.0
+author: >
+  [Patrice Chalin](https://github.com/chalin) ([CNCF](https://www.cncf.io/)),
+  for the [Docsy Steering
+  Committee](/blog/2022/hello/#introducing-the-psc)
+date: 2024-04-30
+# prettier-ignore
+cSpell:ignore: CNCF Chalin subdir
+---
+
+The big news with Docsy [0.10.0] is color themes and dark mode!
+
+## Hugo: breaking changes and deprecation notices
+
+Before jumping into the topic of color modes, I wanted to point out that Docsy's
```

Comment:
Could tighten this up a little bit:

In this release, Docsy's
Hugo dependency is upgraded to **0.125.4** from 0.122.0. An important thing to note here is that Hugo
[0.123.0] was a significant upgrade that included some **breaking changes**.
Before upgrading to this Docsy version, review Hugo's deprecation notices and breaking changes since
0.122.0.

Reply:
Thanks, updated ✔️